cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -6090,6 +6090,13 @@
       Scheduler interval for repository cleanup service, by default 1 hour
     </description>
   </property>
+  <property>
+    <name>source.control.metadata.migration.interval.seconds</name>
+    <value>43200</value>
```

Comment:
We have the capability that if something goes wrong during the migration, it will retry and stop once it is successful. So, shouldn't we have a lesser interval time, say 5 minutes?